cum/scrapers/mangadex.py

```diff
@@ -70,7 +70,7 @@ def name(self):
 
 
 class MangadexChapter(BaseChapter):
-    url_re = re.compile(r'(?:https?://mangadex\.com)?/chapter/([0-9]+)')
+    url_re = re.compile(r'(?:https?://mangadex\.org)?/chapter/([0-9]+)')
```

Comment:
It's probably better to match both tlds here like above with (?:org|com)

Reply:
Thanks, fixed.